examples/insurance/README.md

```diff
@@ -30,6 +39,14 @@ You can finish configuring the flows and run the steps to complete the integrati
 10. Run the `CustomerMaster` flow to master the Advantage and Bedrock customer data. This merges documents for two matching customers in the final database. You can view the results in the Browse Data view.
 
 
+## How to enrich Customer Data using custom step
+
+1. Start the Data Hub and select the project folder: examples/insurance
+2. View the `customEnrichment` flow. Configure the `ingestAdvantage` step by setting the Source Directory Path to the `datasets/advantage` directory (exact path will depend on your filesystem).
+3. The scaffolded custom main.sjs has been edited to insert code that manipulates the instance to insert geo-spatial information corresponding to the postal code
```

Comment:
I recommend including the full path here, as I think most readers won't know where to find this otherwise. And I think you should also explain how you generated the scaffolded main.sjs file, as I don't think we can assume that the reader knows how to do that part either.